itou/scripts/tests.py

```diff
@@ -769,7 +769,7 @@ def test_import_data_into_itou(self):
         )
         input_df = pandas.DataFrame([CleanedAiCsvFileMock()])
         output_df = command.import_data_into_itou(df=input_df, to_be_imported_df=input_df)
-        assert User.objects.filter(is_job_seeker=True).count() == 1
+        assert User.objects.filter(kind="job_seeker").count() == 1
```

Comment:
```suggestion
        assert User.objects.filter(kind=UserKind.JOB_SEEKER).count() == 1
```
? (et pareil juste au dessus)